### ISSUE TITLE
False Positive: POST /wp-json/wc/store/v1/batch

### ISSUE THREAD

**Issue by bergminze** [coreruleset/wordpress-rule-exclusions-plugin#105]
wordpress: Version 6.9.4
woocommerce plugin: Version 10.6.1

The operation to remove a selected item from the shopping cart is blocked by ModSecurity.

Attached you can see the Bunkerweb logs and the ModSecurity rule (932120, 949110) that is blocking the operation.

The log excerpt is attached.

-----------
1) Warning:
-----------
bunkerweb     | 2026/03/26 12:31:01 [warn] 382#382: *1720 ModSecurity: Warning. Matched "Operator `PmFromFile' with parameter `windows-powershell-commands.data' against variable `ARGS:json.requests.array_0.path' (Value: `/wc/store/v1/cart/remove-item' ) 
[file /usr/share/bunkerweb/core/modsecurity/files/coreruleset-v4/rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf"] 
[line "218"] 
[id "932120"] 
[rev ""] 
[msg "Remote Command Execution: Windows PowerShell Command Found"] 
[data "Matched Data: Remove-Item found within ARGS:json.requests.array_0.path: /wc/store/v1/cart/remove-item"] 
[severity "2"] 
[ver "OWASP_CRS/4.24.1"] 
[maturity "0"] 
[accuracy "0"] 
[tag "application-multi"] 
[tag "language-shell"] 
[tag "language-powershell"] 
[tag "platform-windows"] 
[tag "attack-rce"] 
[tag "paranoia-level/1"] 
[tag "OWASP_CRS"] 
[tag "OWASP_CRS/ATTACK-RCE"] 
[tag "capec/1000/152/248/88"] 
[hostname "subdomain.example.org"] 
[uri "/wp-json/wc/store/v1/batch"] 
[unique_id "177452466195.561893"] 
[ref "o18,11v27,29t:cmdLine"], client: 217.88.88.66, server: subdomain.example.org, request: "POST /wp-json/wc/store/v1/batch?_locale=site HTTP/2.0", host: "subdomain.example.org", referrer: "https://subdomain.example.org/warenkorb/

--------------
2) Error/Block
--------------
bunkerweb     | 2026/03/26 12:31:01 [error] 382#382: *1720 [client 217.88.88.66] 
ModSecurity: Access denied with code 403 (phase 2). Matched "Operator `Ge' with parameter `5' against variable `TX:BLOCKING_INBOUND_ANOMALY_SCORE' (Value: `5' )
[file "/usr/share/bunkerweb/core/modsecurity/files/coreruleset-v4/rules/REQUEST-949-BLOCKING-EVALUATION.conf"]
[line "222"]
[id "949110"]
[rev ""]
[msg ""]
[data ""]
[severity "0"]
[ver "OWASP_CRS/4.24.1"]
[maturity "0"]
[accuracy "0"]
[tag "anomaly-evaluation"]
[tag "OWASP_CRS"]
[hostname "subdomain.example.org"]
[uri "/wp-json/wc/store/v1/batch"]
[unique_id "177452466195.561893"]
[ref ""], client: 217.88.88.66, server: subdomain.example.org, request: "POST /wp-json/wc/store/v1/batch?_locale=site HTTP/2.0", host: "subdomain.example.org", referrer: "https://subdomain.example.org/warenkorb/"
bunkerweb     | subdomain.example.org 217.88.88.66 - 6bf7b1b455b8f51c08015ce2ad97b2c5 -
[26/Mar/2026:12:31:01 +0100] "POST /wp-json/wc/store/v1/batch?_locale=site HTTP/2.0" 403 25562 

--------------

[bunkerweb_wordpress_woocommerce_store_v1_batch_operation_blocked.log](https://github.com/user-attachments/files/26273289/bunkerweb_wordpress_woocommerce_store_v1_batch_operation_blocked.log)

**Comment by EsadCetiner** [comment]
Hi @bergminze 

This plugin doesn't support WooCommerce, any false positives you encounter with WooCommerce or any 3rd party plugin must be resolved yourself via an [custom rule exclusion](https://coreruleset.org/docs/2-how-crs-works/2-3-false-positives-and-tuning/).

For this specific false positive, you can write a rule similar to this to allow the request:
```
SecRule REQUEST_FILENAME "@rx /wp-json/wc/store/v[0-9.]+/batch$" \
    "id:<fixme>,\
    phase:1,\
    pass,\
    t:none,\
    nolog,\
    ctl:ruleRemoveTargetById=932120;ARGS:json.requests.array_0.path"
```
